firebird/mocks/driver.py

```diff
@@ -0,0 +1,30 @@
+from firebird.mocks import aardvark as ma
+from firebird import aardvark as a
+from firebird import chip
+from functools import partial
+from firebird.driver import chip_spec_urls, csort, to_rod, to_pyccd, pyccd_dates
+
+
+def pyccd_rdd(specs_url, chips_url, x, y, acquired):
```

Comment:
I totally get the reason for this.  Should we spend some time and think about how to modify the source code to support running n pixels instead of 10,000?  Mocking up major portions of our source code is a bit.... scary.